### ISSUE TITLE
CVE export not possible for multiple projects

### ISSUE THREAD

**Issue by temirg** [goharbor/harbor#18202]
If you are reporting a problem, please make sure the following information are provided:

**Expected behavior and actual behavior:**
Expected: select multiple or all projects => export CVEs should be possible
Tested with an MS AD user from the harbor-wide sysadmin group and with local user: admin.

Actual behavior: Screenshots (user: admin)

one project selected, "export CVEs" is active
![harbor-2 6 3_CVE-export1](https://user-images.githubusercontent.com/112331881/218093649-93e8cf00-e560-43bf-965b-4bf7a70cc20f.png)

two projects selected (or all - it doesn't matter), "export CVEs" is inactive
![harbor-2 6 3_CVE-export2](https://user-images.githubusercontent.com/112331881/218093709-d581346c-81a6-4304-8c2e-36a1d2838bb9.png)

**Versions:**

- harbor version: [2.6.3]
- docker engine version: [20.10.17-ce]
- docker-compose version: [1.29.2]

**Additional context:**
If more details needed - I can gladly supply.

**Comment by wy65701436** [comment]
Yes, from the performance perspective, only one project is allowed to export CVE at the same time. 

Workaround here maybe, you can execute the export multiple times and then merge the results at local. 

Let's keep this issue open as a feature request.

**Comment by titanventura** [comment]
Hey, Is this issue available ? Can I take this up ?
